File: sdks/thunkmetrc-go/wrapper/models/plants_create_plant_plant_batch_packages_request_item.go
package models

type PlantsCreatePlantPlantBatchPackagesRequestItem struct {
    ActualDate string `json:"ActualDate,omitempty"`
    Count int `json:"Count,omitempty"`
    IsDonation bool `json:"IsDonation,omitempty"`
    IsTradeSample bool `json:"IsTradeSample,omitempty"`
    Item string `json:"Item,omitempty"`
    Location string `json:"Location,omitempty"`
    Note string `json:"Note,omitempty"`
    PackageTag string `json:"PackageTag,omitempty"`
    PatientLicenseNumber string `json:"PatientLicenseNumber,omitempty"`
    PlantBatchType string `json:"PlantBatchType,omitempty"`
    PlantLabel string `json:"PlantLabel,omitempty"`
    Sublocation string `json:"Sublocation,omitempty"`
}

File: sdks/thunkmetrc-rs/wrapper/src/models/plants_create_plant_plant_batch_packages_request_item.rs
use serde::{Deserialize, Serialize};
#[allow(unused_imports)]
use super::*;

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct PlantsCreatePlantPlantBatchPackagesRequestItem {
    #[serde(rename = "actual_date", skip_serializing_if = "Option::is_none")]
    pub actual_date: Option<String>,
    #[serde(rename = "count", skip_serializing_if = "Option::is_none")]
    pub count: Option<i64>,
    #[serde(rename = "is_donation", skip_serializing_if = "Option::is_none")]
    pub is_donation: Option<bool>,
    #[serde(rename = "is_trade_sample", skip_serializing_if = "Option::is_none")]
    pub is_trade_sample: Option<bool>,
    #[serde(rename = "item", skip_serializing_if = "Option::is_none")]
    pub item: Option<String>,
    #[serde(rename = "location", skip_serializing_if = "Option::is_none")]
    pub location: Option<String>,
    #[serde(rename = "note", skip_serializing_if = "Option::is_none")]
    pub note: Option<String>,
    #[serde(rename = "package_tag", skip_serializing_if = "Option::is_none")]
    pub package_tag: Option<String>,
    #[serde(rename = "patient_license_number", skip_serializing_if = "Option::is_none")]
    pub patient_license_number: Option<String>,
    #[serde(rename = "plant_batch_type", skip_serializing_if = "Option::is_none")]
    pub plant_batch_type: Option<String>,
    #[serde(rename = "plant_label", skip_serializing_if = "Option::is_none")]
    pub plant_label: Option<String>,
    #[serde(rename = "sublocation", skip_serializing_if = "Option::is_none")]
    pub sublocation: Option<String>,
}
